### ISSUE TITLE
SWIFT_VERSION

### ISSUE THREAD

**Issue by krzyzanowskim** [CocoaPods/CocoaPods#5793]
I created new project with Xcode8 beta6 and added Swift project. After open workspace Xcode assist me with upgrade Swift code and is using Swift 2 instead Swift 3. I found out this is because SWIFT_VERSION is unspecified.

![screen shot 2016-08-28 at 09 03 48](https://cloud.githubusercontent.com/assets/758033/18032227/6f2ec73c-6cff-11e6-80ea-aba7b2238268.png)

Fix is to set it with value from originated framework project

![screen shot 2016-08-28 at 09 12 21](https://cloud.githubusercontent.com/assets/758033/18032237/c3526274-6cff-11e6-8768-fe53dc9ee870.png)

ref. https://github.com/krzyzanowskim/CryptoSwift/issues/314

### Stack

```
   CocoaPods : 1.0.1
        Ruby : ruby 2.0.0p643 (2015-02-25 revision 49749) [x86_64-darwin14.1.1]
    RubyGems : 2.4.6
        Host : Mac OS X 10.11.6 (15G31)
       Xcode : 8.0 (8S201h)
         Git : git version 2.8.4 (Apple Git-73)
Ruby lib dir : /Users/marcinkrzyzanowski/.rvm/rubies/ruby-2.0.0-p643/lib
Repositories : artsy - https://github.com/artsy/Specs.git @ ef8ffbf6d794b7853a662bdc067036382380d45b
               master - https://github.com/CocoaPods/Specs.git @ 375b8228fcddd1fce8cc259894452f90064d4778
```

### Podfile

```ruby
# Uncomment this line to define a global platform for your project
# platform :ios, '9.0'

target 'testttt' do
  # Comment this line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for testttt
  pod 'CryptoSwift', :git => "https://github.com/krzyzanowskim/CryptoSwift", :branch => "swift3"

end
```

## Project that demonstrates the issue

[testttt.zip](https://github.com/CocoaPods/CocoaPods/files/441027/testttt.zip)

**Comment by krzyzanowskim** [comment]
I found `swift_version` in sources, but can't use it. Trying setup in Podfile

```
swift_version = '3.0'
```

but it don't change anything in my case.

**Comment by krzyzanowskim** [comment]
Closing, I found it's addressed in `1.1.0.beta.1`
